src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name="Users")
+public class User{
+	
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    
+    @NotNull
+    private String userName;
+    
+    @NotNull
+    private String name;
+    
+    @NotNull
+    private Date birthDate;
+    
+    @ManyToMany(cascade = {CascadeType.REFRESH, CascadeType.MERGE})
+//    @JoinTable(name = "user_book", 
+//      joinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"), 
+//      inverseJoinColumns = @JoinColumn(name = "book_id", 
+//      referencedColumnName = "id"))
+    private List<Book> books = new ArrayList<Book>();
+
+    
+    public User() {
+    	
+    }
+
+    public long getId() {
+    	return id;
+    }
+
+	public String getUsername() {

```

Comment:
Accommodate the identation of the methods in this class.

Reply:
La tabulación la veo correcta en el eclipse... Ahi lo subo de nuevo a ver que onda